app/controllers/shopify_app/callback_controller.rb

```diff
@@ -24,8 +24,31 @@ def callback
       end
     end
 
+    def jwt_callback
```

Comment:
Since this is only for user tokens, should we call this `user_callback`?

Reply:
I think it's more tightly coupled to the `jwt` aspect than the `user` aspect, since the other callback endpoint can also handle user tokens.